verifiers/proofofsql/src/verifier_should.rs

```diff
@@ -0,0 +1,137 @@
+// Copyright 2024, Horizen Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![cfg(test)]
+
+use rstest::*;
+use sp_core::ConstU32;
+
+use super::*;
+
+struct ConfigWithMaxNuEqualTo5;
+
+impl Config for ConfigWithMaxNuEqualTo5 {
+    type LargestMaxNu = ConstU32<5>;
+}
+
+struct ConfigWithMaxNuEqualTo3;
+
+impl Config for ConfigWithMaxNuEqualTo3 {
+    type LargestMaxNu = ConstU32<3>;
+}
+
+struct TestData {
+    vk: Vec<u8>,
+    proof: Vec<u8>,
+    pubs: Vec<u8>,
+}
+
+#[fixture]
+fn valid_test_data() -> TestData {
+    TestData {
+        vk: include_bytes!("resources/VALID_VK.bin").to_vec(),
+        proof: include_bytes!("resources/VALID_PROOF.bin").to_vec(),
+        pubs: include_bytes!("resources/VALID_PUBS.bin").to_vec(),
+    }
+}
+
+#[rstest]
+fn verify_valid_proof(valid_test_data: TestData) {
+    assert!(ProofOfSql::<ConfigWithMaxNuEqualTo5>::verify_proof(
```

Comment:
`assert_ok` from `frame_support` is quite nicer

Reply:
Done in https://github.com/HorizenLabs/zkVerify/pull/136/commits/2e24f51b7446f657dc81b1018abf745f74bd6a64